packages/functions/src/web.ts

```diff
@@ -48,7 +50,7 @@ export class FirebaseFunctionsWeb
   }
 
   public async useEmulator(options: UseEmulatorOptions): Promise<void> {
-    const functions = getFunctions();
+    const functions = getFunctions(getApp(), options.regionOrCustomDomain);
```

Comment:
```suggestion
    const functions = getFunctions(undefined, options.regionOrCustomDomain);
```

Is the `getApp()` call necessary? If we set undefined it should automatically take the default app (I haven't tested it, just a thought).

Reply:
Yes, absolutely true, @robingenz. I hadn’t noticed that this parameter was optional. Indeed, without defining it, it automatically takes the default app.